### PR TITLE
Openapi frontend

### PR DESCRIPTION
**Pull request by Latz3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,7 @@
 # oTTo
 open Time Table organizer
+
+
+Build
+
+`docker run -it --rm --name otto -v "$(pwd)":/usr/src/mymaven -w /usr/src/mymaven maven:3.9-eclipse-temurin-17 mvn clean install`
```
Comment (kaidobit) [inline README.md:7]: please use maven wrapper

File: packages/frontend/pom.xml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                        <execution>
+                        <id>Build OpenAPI code</id>
+                        <goals>
+                            <goal>npm</goal>
+                        </goals>
+
+                        <!-- optional: default phase is "generate-resources" -->
+                        <phase>generate-resources</phase>
+
+                        <configuration>
```
Comment (kaidobit) [inline packages/frontend/pom.xml:71]: do we need something else then install?
yes -> remove comment
no -> remove arguments-tag

File: packages/frontend/pom.xml [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+                                <output>${project.basedir}/generated-sources/client</output>
+                            </configuration>
+                        </execution>
+                    </executions>
+                </plugin>
+                <plugin>
+                    <groupId>com.github.eirslett</groupId>
+                    <artifactId>frontend-maven-plugin</artifactId>
+                    <!-- Use the latest released version:
+                    https://repo1.maven.org/maven2/com/github/eirslett/frontend-maven-plugin/ -->
```
Comment (kaidobit) [inline packages/frontend/pom.xml:50]: remove comment

File: packages/frontend/pom.xml [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+                    <version>1.12.1</version>
+                    <executions>
+                        <execution>
+                            <!-- optional: you don't really need execution ids, but it looks nice in your build log. -->
```
Comment (kaidobit) [inline packages/frontend/pom.xml:54]: remove comment

File: packages/frontend/pom.xml [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+                            <id>install node and npm</id>
+                            <goals>
+                                <goal>install-node-and-npm</goal>
+                            </goals>
+                            <!-- optional: default phase is "generate-resources" -->
```
Comment (kaidobit) [inline packages/frontend/pom.xml:59]: remove comment

**Comment by kaidobit** [comment]
please add the link to the `typescript-axio`-generator-documentation to `docs/usefullinks:md`

File: packages/frontend/pom.xml (hunk 2 above)
Comment (Hustenbonbon) [reply packages/frontend/pom.xml:71]: As the person who is most likely to work with that in the future, I find it helpful to keep the comments

File: packages/frontend/pom.xml (hunk 3 above)
Comment (Hustenbonbon) [reply packages/frontend/pom.xml:50]: As the person who is most likely to work with that in the future, I find it helpful to keep the comments

**Pull request merged**